<launch>
  <param name="robot_description" textfile="$(find phoebe_description)/phoebe.urdf"/>

  <node name="phoebe_state_cfg" pkg="rostopic" type="rostopic" args="pub -1 /jointmap phoebe/JointMap &quot;{tf_joints: [{data: 'wheel_right_joint'}, {data: 'wheel_left_joint'}, {data: 'ptu_tilt_joint'}, {data: 'ptu_pan_joint'}, {data: 'right_arm_shoulder_roll_joint'}, {data: 'right_arm_shoulder_pitch_joint'}, {data: 'right_arm_shoulder_yaw_joint'}, {data: 'right_arm_elbow_pitch_joint'}, {data: 'right_arm_elbow_yaw_joint'}, {data: 'right_arm_wrist_pitch_joint'}, {data: 'right_arm_wrist_roll_joint'}, {data: 'right_arm_gripper_joint'}, {data: 'right_arm_gripper_joint2'}, {data: 'left_arm_shoulder_roll_joint'}, {data: 'left_arm_shoulder_pitch_joint'}, {data: 'left_arm_shoulder_yaw_joint'}, {data: 'left_arm_elbow_pitch_joint'}, {data: 'left_arm_elbow_yaw_joint'}, {data: 'left_arm_wrist_pitch_joint'}, {data: 'left_arm_wrist_roll_joint'}, {data: 'left_arm_gripper_joint'}, {data: 'left_arm_gripper_joint2'}], feedback_joints: [{data: 'wheel_right'}, {data: 'wheel_left'}, {data: '/ptu/tilt'}, {data: '/ptu/pan'}, {data: '/right_arm_shoulder_roll_joint/pos'}, {data: 'right_arm_shoulder_pitch_joint/pos'}, {data: 'right_arm_shoulder_yaw_joint/pos'}, {data: 'right_arm_elbow_pitch_joint/pos'}, {data: 'right_arm_elbow_yaw_joint/pos'}, {data: 'right_arm_wrist_pitch_joint/pos'}, {data: 'right_arm_wrist_roll_joint/pos'}, {data: 'right_arm_gripper_joint/pos'}, {data: 'right_arm_gripper_joint/pos'}, {data: 'left_arm_shoulder_roll_joint/pos'}, {data: 'left_arm_shoulder_pitch_joint/pos'}, {data: 'left_arm_shoulder_yaw_joint/pos'}, {data: 'left_arm_elbow_pitch_joint/pos'}, {data: 'left_arm_elbow_yaw_joint/pos'}, {data: 'left_arm_wrist_pitch_joint/pos'}, {data: 'left_arm_wrist_roll_joint/pos'}, {data: 'left_arm_gripper_joint/pos'}, {data: 'left_arm_gripper_joint/pos'}]}&quot;" />

  <node name="phoebe_state_publisher" pkg="phoebe" type="jointstate_publisher" output="screen" />

</launch>
